### ISSUE TITLE
Actors Tab - Folders and Buttons

### ISSUE THREAD

**Issue by RandomLegend** [mouse0270/fantasy-rpg-ui#24]
Hello,

first of all - Great theme! Really love it!

Would it be possible to have the buttons and folders themed aswell? Create Actors (and create Folder for the GM UI)

![image](https://user-images.githubusercontent.com/38812603/219705785-5dd78ae3-984c-4ac2-878b-9f65b4a226e0.png)

As you can see right now it doesn't. It colors them, the way it's set in libthemer, but i'd love to have them those fancy buttons and borders just as everything else has.

**Comment by mouse0270** [comment]
Interesting, I wonder if something is overriding something else or did I honestly miss theming those.

I can take a look, but it will be a few days. My M.2 drive on my PC died and I lost everything, so I am a few months behind on changes I made but never pushed to github like an idiot.

My new drive comes tomorrow or sunday and I gotta get everything setup again.

**Comment by RandomLegend** [comment]
oof thats hard to hear...i'm sorry for the lost work :-/

might want to consider some external backup solution that runs automatically? Or specifically for github code you could automatically upload it to a private repo or fork and if it goes well you can send it to the main repo.

**Comment by mouse0270** [comment]
Wellllllllllllllll

I have external drives and even pay for unlimited google storage.... The problem is, I am stupid and lazy. haha

Edit: Basically I have all the tools and knowledge to prevent this problem... but didn't use them, so no one to blame but me. Advantage is. M.2 drives were super cheap on Wednesday I picked up a 2TB Samsung Pro for 150$

**Comment by RandomLegend** [comment]
2TB for 150$ is really nice! 

But beware of these Samsung Pro M.2 - There are some faulty ones going around that degrade very quickly. Gotta check that before.

Alright, so i just wait until you recovered lol. I assume theming all this would be a very easy task for you :-)

**Comment by mouse0270** [comment]
Yeah, the actual theming of it should be pretty quick. I thought it was themed, which is what has me confused. The only thing I can think of is, the only thing I can think of why I might not have themed it, is because using an image would make setting the colors of the folders pointless. And since I use folder colors to help me organize, I might have left it as is, because it worked better for me.

But honestly, it should be easy to throw a check box to allow users to override it with a themed option.

Is that so? I've had a samsung pro for 4 years now and its been great... Well I guess technically it should have lasted more than four years... So maybe I shouldn't have gotten another samsung one. haha

**Comment by RandomLegend** [comment]
"But honestly, it should be easy to throw a check box to allow users to override it with a themed option."

yeah that would be perfect i think. I usually use the colors aswell, but as of right now we don't use foundry as a typical VTT but rather to replace DnD Beyond. So as of right now, everyone just has a folder with their name to put their chars into. And is the GM just put my enemie actors in a different folder which i clear after a session.

___

"Is that so? I've had a samsung pro for 4 years now and its been great... Well I guess technically it should have lasted more than four years... So maybe I shouldn't have gotten another samsung one. haha" No it only affects the newest models. There was something on LTT aswell. Check it out, maybe it's fixed again i don't know

**Comment by RandomLegend** [comment]
Hey mate, any updates on your drive? :-D

**Comment by mouse0270** [comment]
Yup, I got foundry installed on Tuesday, but have been busy with work. Should be looking into this over the weekend.

**Comment by RandomLegend** [comment]
Ah fantastic, just wanted to check in.
Don't stress yourself over this^^
